go.mod

```diff
@@ -3,49 +3,56 @@ module github.com/blackducksoftware/synopsys-operator
 go 1.13
 
 require (
-	github.com/Azure/go-autorest/autorest v0.8.0 // indirect
+	github.com/Azure/go-autorest v13.3.3+incompatible // indirect
+	github.com/Azure/go-autorest/autorest/adal v0.8.2 // indirect
 	github.com/blackducksoftware/horizon v0.0.0-20190625151958-16cafa9109a3
 	github.com/docker/spdystream v0.0.0-20181023171402-6480d4af844c // indirect
 	github.com/elazarl/goproxy v0.0.0-20190911111923-ecfe977594f1 // indirect
-	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
 	github.com/gin-gonic/gin v1.4.0
+	github.com/gobuffalo/envy v1.7.1 // indirect
 	github.com/gobuffalo/packr v1.30.1
-	github.com/google/go-cmp v0.3.0
+	github.com/google/go-cmp v0.4.0
+	github.com/google/gofuzz v1.1.0 // indirect
 	github.com/googleapis/gnostic v0.3.1 // indirect
 	github.com/gophercloud/gophercloud v0.3.0 // indirect
 	github.com/gorilla/handlers v1.4.2
 	github.com/gorilla/mux v1.7.3
-	github.com/hashicorp/golang-lru v0.0.0-20180201235237-0fb14efe8c47 // indirect
 	github.com/imdario/mergo v0.3.7
 	github.com/juju/errors v0.0.0-20190806202954-0232dcc7464d
 	github.com/juju/loggo v0.0.0-20190526231331-6e530bcce5d8 // indirect
 	github.com/juju/testing v0.0.0-20190723135506-ce30eb24acd2 // indirect
 	github.com/lib/pq v1.2.0
+	github.com/mattn/go-runewidth v0.0.4 // indirect
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/onsi/ginkgo v1.8.0
-	github.com/onsi/gomega v1.5.0
-	github.com/openshift/api v3.9.0+incompatible
-	github.com/openshift/client-go v3.9.0+incompatible
-	github.com/pkg/errors v0.8.1
+	github.com/onsi/ginkgo v1.10.1
+	github.com/onsi/gomega v1.7.0
+	github.com/openshift/api v0.0.0-20200217161739-c99157bc6492
+	github.com/openshift/client-go v0.0.0-20200116152001-92a2713fa240
+	github.com/pkg/errors v0.9.1
+	github.com/rogpeppe/go-internal v1.4.0 // indirect
 	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/cobra v0.0.5
-	github.com/spf13/pflag v1.0.3
+	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.4.0
-	github.com/stretchr/testify v1.3.0
-	gopkg.in/inf.v0 v0.9.1 // indirect
+	github.com/stretchr/testify v1.4.0
+	golang.org/x/net v0.0.0-20200202094626-16171245cfb2 // indirect
+	google.golang.org/appengine v1.6.5 // indirect
 	gopkg.in/mgo.v2 v2.0.0-20190816093944-a6b53ec6cb22 // indirect
-	k8s.io/api v0.0.0-20190808180749-077ce48e77da
-	k8s.io/apiextensions-apiserver v0.0.0-20190809061809-636e76ffcf57
-	k8s.io/apimachinery v0.0.0-20190809020650-423f5d784010
-	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
-	k8s.io/klog v0.4.0 // indirect
-	k8s.io/kube-openapi v0.0.0-20190722073852-5e22f3d471e6 // indirect
-	k8s.io/utils v0.0.0-20190809000727-6c36bc71fc4a // indirect
+	helm.sh/helm/v3 v3.1.0-rc.1.0.20200219163831-598dc504d634
+	k8s.io/api v0.17.3
+	k8s.io/apiextensions-apiserver v0.17.2
+	k8s.io/apimachinery v0.17.3
+	k8s.io/cli-runtime v0.17.2
+	k8s.io/client-go v0.17.2
+	k8s.io/helm v2.16.3+incompatible
```

Comment:
are we supporting both helm v2 and v3?

Reply:
Senthil explained to me, just noting it down for reference.  We're importing helm v2 part of the library as part of the copy/paste to support rendering.  See https://github.com/blackducksoftware/synopsys-operator/pull/1234/files#diff-3272523575a3b8916bbf7254ca930b29R194

Reply:
Actually, there is a helm v3 version of this, so we should try and minimize our dependency to just v3: helm.sh/helm/v3/pkg/releaseutil